cdap-ui/app/cdap/components/FieldLevelLineage/store/Store.js

```diff
@@ -35,7 +36,8 @@ const defaultInitialState = {
   fields: [],
   backward: [],
   activeField: null,
-  search: ''
+  search: '',
+  timeSelection: 'last7d'
```

Comment:
Can you try `Object.keys(TIME_OPTIONS][0]` instead of hardcoding `last7d`? Not sure if the same ordering of keys will be maintained though